5.application_scopes.md

```diff
@@ -142,48 +165,60 @@ The health scope is defined by the following spec:
 
 ```yaml
 apiVersion: core.oam.dev/v1alpha1
-kind: ApplicationScope
+kind: ScopeDefinition
 metadata:
-  name: health
-  annotations:
-    version: v1.0.0
-    description: "aggregated health state for a group of components."
+  name: healthscope.core.oam.dev
 spec:
-  type: core.oam.dev/v1alpha1.HealthScope
   allowComponentOverlap: true
-  parameters:
-    - name: probe-method
-      description: The method to probe the components, e.g. 'httpGet'.
-      type: string
-      required: true
-    - name: probe-endpoint
-      description: The endpoint to probe from the components, e.g. '/v1/health'.
-      type: string
-      required: true
-    - name: probe-timeout
-      description: The amount of time in seconds to wait when receiving a response before marked failure.
-      type: integer
-      required: false
-    - name: probe-interval
-      description: The amount of time in seconds between probing tries.
-      type: integer
-      required: false
-    - name: failure-rate-threshold
-      description: If the rate of failure of total probe results is above this threshold, declared 'failed'.
-      type: double
-      required: false
-    - name: healthy-rate-threshold
-      description: If the rate of healthy of total probe results is above this threshold, declared 'healthy'.
-      type: double
-      required: false
-    - name: health-threshold-percentage
-      description: The % of healthy components required to upgrade scope
-      type: double
-      required: false
-    - name: required-healthy-components
-      description: Comma-separated list of names of the components required to be healthy for the scope to be health.
-      type: []string
-      required: false
+  definitionRef:
+    name: healthscope.core.oam.dev
+    version: v1alpha1
+    schematic: |
```

Comment:
According to https://github.com/oam-dev/spec/pull/304/, the scope definitionRef will point to a CRD name. We don't need to embed json schema here, do we?